Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:9 as builder
+
+# Builds cognite-grafana-datasource
+WORKDIR /app
+
+COPY . /app
+RUN yarn
+RUN yarn build
+
+FROM grafana/grafana
```

Comment:
Add a tag, `:5.2.3` since we don't have guarantees about backwards compatibility for plugins afaik

Reply:
Good point